src/dev/i18n_tools/bin/run_i18n_check.ts

```diff
@@ -131,13 +128,15 @@ run(
       {
         concurrent: false,
         exitOnError: true,
-        renderer: process.env.CI ? 'verbose' : ('default' as any),
+        forceTTY: false,
+        renderer:
+          ((silent || quiet) && 'silent') || (process.env.CI ? 'verbose' : ('default' as any)),
```

Comment:
Why 2 flags that do the same thing? `silent` and `quiet`
i think lets just keep the `quiet` flag.

To get rid of the any type we can just add the renderers to the constructor type on Line 90:
```
- const list = new Listr<I18nCheckTaskContext>(
+ const list = new Listr<I18nCheckTaskContext, 'default' | 'verbose'>(
```

We can simplify this
```
renderer: ((silent || quiet) && 'silent') || (process.env.CI ? 'verbose' : ('default' as any))
renderer: quiet? 'silent' : process.env.CI? 'verbose' : 'default';
```

Reply:
`--silent` and `--quiet` are both pre-setup for any CLI script ran through the `run` function from the dev-cli-runner. In our case, they both shoot for the same thing, so I thought projected to this aspect, either of the flags will be useful.

I'd not rely solely on the `--silent` flag, because that quenches any output (including error messages) from the runner, whereas `--quiet` keeps output to the minimum, which I think we should do here.